Library/Homebrew/dev-cmd/extract.rb

```diff
@@ -125,11 +125,12 @@ def extract
     if args.version
       ohai "Searching repository history"
       version = args.version
-      rev = "HEAD"
+      rev = nil
```

Comment:
Why does this need to change?

Reply:
```ruby
rev, (path,) = Git.last_revision_commit_of_files(repo, pattern, before_commit: "#{rev}~1")
```
Above was the original `rev` looping code. The `HEAD` would get skipped on the first time it ran.